libs/designsystem/src/lib/components/data-table/thead/thead.component.scss

```diff
@@ -0,0 +1,16 @@
+@use '~@kirbydesign/core/src/scss/utils';
+@use '../data-table.utils';
+
+:host-context([kirby-thead]) {
+  @include utils.slotted([kirby-tr]) {
+    height: utils.size('l');
+    text-align: left;
+
+    @include data-table.border-bottom;
+
+    @include utils.slotted(th) {
+      @include data-table.header-typgraphy;
```

Comment:
This seems like something that should be owned by the component it's used in. Generally just makes it easier to refactor and maintain, though I see the benefit of having table component as the master.